apis/fluentbit/v1alpha2/plugins/output/azure_blob_types.go

```diff
@@ -0,0 +1,74 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// Azure Blob is the Azure Blob output plugin, allows to ingest your records into Azure Blob Storage.
+type AzureBlob struct {
+	// Azure Storage account name
+	AccountName string `json:"account_name,omitempty"`
```

Comment:
The `account_name` is  mandatory, should not be `omitempty`.

Same as `shared_key` and `container_name`.

Reply:
`AccountName string `json:"account_name,omitempty"`` should be `AccountName string `json:"accountName,omitempty"``

Reply:
>AccountName string json:"account_name,omitempty" should be
AccountName string json:"accountName,omitempty"

Was wondering what the rule here is? From my (naive) understanding, I looked at DataDog output plugins and they match what the fluentbit documentation is (see [apiKey](https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/plugins/output/datadog_types.go#L23) for datadog in [fluentbit](https://docs.fluentbit.io/manual/pipeline/outputs/datadog#configuration-parameters), but I've just looked at OpenSearch and Splunk now and maybe that was the exception rather than the rule.

Will change, just curious on whether this is a naming convention thing or I'm misunderstanding how it works 👍🏻

Reply:
> Will change, just curious on whether this is a naming convention thing or I'm misunderstanding how it works

@HarleyB123 It's kind of [K8s' conversion](https://v1-22.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#podspec-v1-core) and fluent-operator follows that. All plugins use `camelCase` naming convention https://github.com/fluent/fluent-operator/tree/master/apis/fluentbit/v1alpha2/plugins/output

Reply:
Changed in https://github.com/fluent/fluent-operator/pull/549/commits/b6a85d68edb15dd9d6e63ebde5cc62099b9c66af